### PR TITLE
Don't run CI checks if only the site changed

### PR DESCRIPTION
**Pull request by nrb**
Also include the github directory to enable faster iteration on GitHub
actions

Signed-off-by: Nolan Brubaker <brubakern@vmware.com>

### CODE REVIEW

**Review by carlisia — DISMISSED** [review]
Awesome 🎉

**Comment by jenting** [comment]
Related to https://github.com/vmware-tanzu/velero/issues/2557

**Comment by jenting** [comment]
This needs to address the conflicts.

**Comment by nrb** [comment]
Conflicts fixed!

File: .github/workflows/pr-changelog-check.yml [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+      - "site/**"
+      - ".github/**"
+      - "design/**"
 jobs:
-
   build:
     name: Run Changelog Check
     runs-on: ubuntu-latest
     steps:
+      - name: Check out the code
```
Comment (zubron) [inline .github/workflows/pr-changelog-check.yml:14]: This step is being included twice now.

**Review by zubron — CHANGES_REQUESTED** [review]
This is great, thanks @nrb! There's a single fix needed in the changelog check action where code checkout is being called twice.

**Review by jenting — APPROVED** [review]

**Review by zubron — APPROVED** [review]

**Comment by zubron** [comment]
Hmm, I don't know if this will work in it's current state. We have branch protection for main that requires that the `Run CI` check must pass before merging, but this will now never run on PRs like this. I don't know if there is a way to make the checks conditional. Looks like it's an open request in the GitHub community: https://github.community/t/feature-request-conditional-required-checks/16761

**Comment by nrb** [comment]
> We have branch protection for main that requires that the Run CI check must pass before merging, but this will now never run on PRs like this.

Oof, you're right. So we'd need to make the change in the Run CI workflow to check for this. I _think_ we can do that.

**Comment by nrb** [comment]
I'm going to close this and look into an alternate approach. I have another idea for cleaning up some of the CRD testing anyway, so that they're consolidated into 1 file.

**Pull request closed without merging**